### PR TITLE
Fix image loading in normalMap sample

### PR DESCRIPTION
**Pull request by cmhhelgeson**
Get the image loading in the normalMap sample to actually work.

### CODE REVIEW

**Comment by cmhhelgeson** [comment]
How are assets provided to the static build of the Next project? Adding images to assets and then running build or export doesn't seem to add them to either the static build or the out build.

**Comment by cmhhelgeson** [comment]
Got rid of any utils outside of the main function that handled loading images. I also put all the images back in the assets folder. For some reason, the .next folder and out folder only build in the assets as part of the website when the images are referenced in main? Not sure why that might be. Regardless, image loading should be fixed after this next commit.

**Comment by cmhhelgeson** [comment]
This should probably be addressed as soon as possible, the current example on the website is nonfunctional.

File: next.config.js [hunk 1]
```diff
@@ -8,6 +8,9 @@ module.exports = {
   basePath: BASE_PATH,
   compress: true,
   reactStrictMode: true,
+  eslint: {
```
Comment (greggman) [inline next.config.js:11]: remove?

File: src/sample/normalMap/main.ts [hunk 2]
```diff
@@ -219,7 +284,7 @@ const init: SampleInit = async ({ canvas, pageState, gui }) => {
   function getModelMatrix() {
     const modelMatrix = mat4.create();
     mat4.identity(modelMatrix);
-    mat4.rotateX(modelMatrix, 10, modelMatrix);
+    mat4.rotateX(modelMatrix, 0, modelMatrix);
```
Comment (greggman) [inline src/sample/normalMap/main.ts:287]: why rotate by 0? remove?

File: src/sample/texturedCube/main.ts [hunk 3]
```diff
@@ -110,6 +110,11 @@ const init: SampleInit = async ({ canvas, pageState }) => {
   // Fetch the image and upload it into a GPUTexture.
   let cubeTexture: GPUTexture;
   {
+    const url = new URL(
```
Comment (greggman) [inline src/sample/texturedCube/main.ts:113]: why is this changed? revert?

**Review by greggman — CHANGES_REQUESTED** [review]

File: src/sample/normalMap/main.ts (hunk 2 above)
Comment (cmhhelgeson) [reply src/sample/normalMap/main.ts:287]: Fixed

File: next.config.js (hunk 1 above)
Comment (cmhhelgeson) [reply next.config.js:11]: Removed

**Review by greggman — APPROVED** [review]

**Pull request merged**